WebHosts/WebHostManager/WebHostManager.csproj

```diff
@@ -24,6 +24,7 @@
         <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
         <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
         <PackageReference Include="Serilog.AspNetCore" Version="8.0.2" />
+        <PackageReference Include="Serilog.Settings.AppSettings" Version="3.0.0" />
```

Comment:
What's this change for?

Reply:
WebHostManager doesn't start when it's not included